apps/synapse-portal-framework/src/components/EntityPageMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import { DetailsPageSectionLayoutType } from '@/components/DetailsPage/DetailsPageSectionLayout'
+import { scrollToWithOffset } from '@/utils'
+import { Button, Stack, Typography } from '@mui/material'
+import { NavigateFunction, Location } from 'react-router'
+
+const EntityPageMenu = ({
```

Comment:
Prefer to rename this `DetailsPageMenu`, since it is used in the `DetailsPage` (in my mind, this new entity page is a kind of details page, but not all details pages are Entity pages)